downstream/modules/automation-hub/proc-obtain-images.adoc

```diff
@@ -0,0 +1,46 @@
+
+
+[id="obtain-images"]
+
+
+= Obtaining images for use in {HubName}
+
+
+[role="_abstract"]
+Before you can push container images to your private {HubName}, you must first pull them from an existing registry and tag them for use. This example details how to pull an image from the Red Hat Ecosystem Catalog (registry.redhat.io).
+
+.Prerequisites
+
+* You have permissions to pull images from registry.redhat.io
+
+.Procedure
+
+. Log in to Podman using your registry.redhat.io credentials:
++
+-----
+$ podman login `registry.redhat.io`
+-----
++
+. Enter your username and password at the prompts.
+. Pull a container image:
++
+-----
+$ podman pull registry.redhat.io/[container image name]:[tag]
+-----
+
+
+.Verification
+
+
+. List the images in local storage:
++
+-----
+$ podman images
+-----
++
+. Verify that the image you recently pulled is contained in the list.
```

Comment:
I looked at the podman docs - does the images command display the tags or does the command need a flag?